react-native/react/native/remote-component-loader.js

```diff
@@ -49,6 +50,14 @@ class RemoteComponentLoader extends Component {
 
     const componentToLoad = getQueryVariable('component')
     const selectorParams = getQueryVariable('selectorParams')
+    let title = getQueryVariable('title')
```

Comment:
I just mean we pass all params to bootstrap the thing with the query parameters so unless we pass title in getParams later (?). We can send both